scalyr_agent/__scalyr__.py

```diff
@@ -139,9 +139,14 @@ def get_install_root():
 def __add_scalyr_package_to_path():
     """Adds the path for the scalyr package and embedded third party packages to the PYTHONPATH.
     """
-    # prepend the third party directory first so it appears after the package root
+    # prepend the third party directory first so it appears after the package root and
+    # third_party_tls directories
     sys.path.insert(0, os.path.join(get_package_root(), 'third_party'))
 
+    # prepend the third party tls directory first so it appears after the package root
```

Comment:
Should we disable this for windows?  It doesn't get included in the py2exe translation.. it would seem like we could get a weird bug if we somehow reference one of these files while running in Windows.